ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/MultiLineIfElseRule.kt

```diff
@@ -22,14 +27,31 @@ class MultiLineIfElseRule : Rule("multiline-if-else") {
             if (!node.treePrev.textContains('\n')) { // if (...) <statement>
                 return
             }
+
             if (node.firstChildNode?.firstChildNode?.elementType != LBRACE) {
                 emit(node.firstChildNode.startOffset, "Missing { ... }", true)
-                // Disabled until we autocorrect with proper formatting
-                /*if (autoCorrect) {
-                    (node.firstChildNode.firstChildNode as TreeElement).rawInsertBeforeMe(LeafPsiElement(RBRACE, "{"))
-                    (node.lastChildNode.lastChildNode as TreeElement).rawInsertAfterMe(LeafPsiElement(LBRACE, "}"))
-                }*/
+                if (autoCorrect) {
+                    autocorrect(node)
+                }
             }
         }
     }
+
+    private fun autocorrect(node: ASTNode) {
+        node.treeParent.replaceChild(node.treePrev, PsiWhiteSpaceImpl(" "))
+        val previousChild = node.firstChildNode
+        KtBlockExpression(null).apply {
```

Comment:
hm, does this support indentation? For instance, if I had a nested if blocks, like:
```
if (condition1) {
    if (condition2) return true
}
```
I believe the `addChild(previousChild)` will not indent the `return true` block properly.. as well as the following RBRACE

Reply:
I just pushed a change that should indent the code properly.